support-modules/aws/src/main/scala/com/gu/aws/aws.scala

```diff
@@ -14,6 +15,7 @@ package object aws {
     new ProfileCredentialsProvider(ProfileName),
     new InstanceProfileCredentialsProvider(false),
     new EnvironmentVariableCredentialsProvider(),
+    new EC2ContainerCredentialsProviderWrapper(), // for use with lambda snapstart
```

Comment:
see https://docs.aws.amazon.com/lambda/latest/dg/snapstart-activate.html#snapstart-credentials for how credentials are loaded with snapstart